core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -415,7 +415,7 @@ public synchronized InetSocketAddress getConfAddress() throws UnavailableExcepti
           throw se;
         }
       }
-      LOG.debug("Rpc failed ({}): {}", retryPolicy.getAttemptCount(), ex.toString());
+      LOG.debug("Rpc failed ({}): {}", retryPolicy.getAttemptCount(), ex);
```

Comment:
```suggestion
      LOG.debug("Rpc failed ({}): ", retryPolicy.getAttemptCount(), ex);
```

On debug logging, when last argument is Throwable, we don't use `{}` in logging message to force logging the stack trace.
https://docs.alluxio.io/os/user/edge/en/contributor/Code-Conventions.html#debug-log-level